docs/source/AdministratorGuide/Systems/WorkloadManagement/InputDataResolution.rst

```diff
@@ -0,0 +1,35 @@
+.. _InputDataResolution:
+
+==================================================
+InputDataResolution: giving job access to the data
+==================================================
+
+When a job needs access to data, there are two ways data can be accessed:
+
+* either by downloading the file on the local worker node
+* or by reading the data remotely, aka ``streaming``.
+
+The resolution is done in the ``JobWrapper`` (see :ref:`jobs`). By default, the resolution logic is implemented in :py:mod:`~DIRAC.WorkloadManagementSystem.Client.InputDataResolution`. It can be overwritten by the Job JDL (see `InputDataModule` in :ref:`jdlDescription`), or by the ``/Operations/<>/InputDataPolicy/InputDataModule`` parameter.
```

Comment:
`jdlDescription` is not found as reference

Reply:
I don't quite get why, since it is here 
https://github.com/DIRACGrid/DIRAC/pull/6340/files#diff-0077acc6638615da9321dcd2085fad71a14870253bc4af83b5e32a7834b919f0R1

Let's see if the new build solves anything

Reply:
https://stackoverflow.com/a/32358779

So `:ref:` like this only works if the target is directly before a section title, if not title the link needs to get a printable name. Or you need to move the target down to just before the section start.

Reply:
oh ! I did not get such a nice explanation message as the guy on stackoverflow though. Let's try that. Thanks !

Reply:
seems to have worked, thanks !